json_schemas/test_story.schema.yaml

```diff
@@ -190,15 +190,23 @@ definitions:
     type: object
     properties:
       status:
-        type: integer
-        description: The expected HTTP status code. Default to 200.
-        default: 200
+        anyOf:
```

Comment:
It's one or the other, and only one, so I think `oneOf` is more appropriate.

Reply:
The return doesn't matches with the elements of array. it has content type too but that we have already passed in the schema.
```
ERROR   flow_framework/workflow/provision.yaml (tests/default/flow_framework/workflow/provision.yaml)
    ERROR   CHAPTERS
        ERROR   Provision workflow.
            PASSED  PARAMETERS
                PASSED  workflow_id
            PASSED  REQUEST BODY
            ERROR   RESPONSE STATUS (Expected status 200,202, but received 200: application/json.)
            SKIPPED RESPONSE PAYLOAD BODY
            SKIPPED RESPONSE PAYLOAD SCHEMA
            SKIPPED RESPONSE OUTPUT VALUES

Reply:
Had to add logic to handle array of status code for chapters. Thanks @nhtruong for the pointer here. Tests passed. @dblock @dbwiddis @nhtruong should be good to review now. Thanks